python/vend.xml

```diff
@@ -0,0 +1,13 @@
+<Envelope xmlns:ns2="http://webservices.ems.momas.com/" xmlns:ns3="http://schemas.xmlsoap.org/soap/envelope/">
```

Comment:
@zaycodes It seems you accidentally added a wrong file to this pull request :)

Reply:
I've fixed this